src/AdminAcceptanceTests.TestData/User.cs

```diff
@@ -192,11 +194,11 @@ public User RetrieveRandomBuyerUser(string connectionString)
                       ,[FirstName]
                       ,[LastName]
```

Comment:
Nasty (commas at start of line) 😃

Ignoring my particular dislike of that style, it would be more consistent with formatting elsewhere to use commas at the end of lines. Would be better without the square brackets too. 😃 

However, as it's not actually something you've changed as part of this PR or you're particularly passionate about keeping it how it is I won't mind if you leave it as is.

Reply:
I would usually have the commas at the end and no square brackets, but that was written by Tony (I've not had to look at this bit since he left). I'll be adding a few tidy up PR's from all the repos at some point, but it is not today's job

Reply:
Formatting to be improved later.